pkg/crc/cache/cache_darwin.go

```diff
@@ -9,5 +11,13 @@ func NewVfkitCache() *Cache {
 }
 
 func getVfkitVersion(executablePath string) (string, error) {
-	return getVersionGeneric(executablePath, "-v")
+	version, err := getVersionGeneric(executablePath, "-v")
+	if err != nil {
+		return version, err
+	}
+	version = strings.TrimSpace(version)
```

Comment:
Is this required? I don’t think the version check would have worked in the past if there were extra spaces.